analytics/var_importance.md

```diff
@@ -0,0 +1,49 @@
+# Variable Importance
+
+Calculates Variable Importance with Random Forest.
+
+## Input Data
+Input data should contain at least one categorical or numeric column for "What to Predict" and more than one categorical and/or numeric columns as Variable Columns.
+
+  * What to Predict - Numeric or Categorical column that you want to Predict.
+  
+  * Variable Columns - Numeric and/or Categorical columns that you want to check importance to predict your "What to Predict" column.
+
+## How to Use This Feature
+1. Click Analytics View tab.
+2. If necessary, click "+" button on the left of existing Analytics tabs, to create a new Analytics.
+3. Select "Variable Importance" for Analytics Type.
+4. Select What to Predict Column.
+5. Click Variable Columns and open Column Selector Dialog.
+
+![](images/var_importance_column_select.png)
+
+6. Select Columns that you want to see importance.
+7. Click Run button to run the analytics.
+8. Select view type (explained below) by clicking view type link to see each type of generated visualization.
+
+### "Importance" View
+"Importance" View displays importance information on Bar chart with Mean Decrease Gini. The higher the Mean Decrease Gini, more important the variable is. 
+
+![](images/var_importance.png)
+
+### "Importance Table" View
+"Importance Table" View displays the importance in table format with color as indicator of importance. By clicking column header for Importance, you can sort data.
+
+![](images/var_importance_table.png)
+
+### "Model Quality" View
+"Model Quality" View displays the quality of the model created for this Variable Importance Analytics. Each row shows the model performance from the point whether the Class (i.e. Value in What to Predict column) prediction was correct or not if What to Predict is categorical column.
```

Comment:
There should be explanation for the case when the target column is numeric.

Reply:
Added explanation for a case when What to Predict column is numeric column.